tests/integration-service/README.md

```diff
@@ -0,0 +1,154 @@
+# Integration Service
+
+The Integration Service manages the lifecycle, deployment and integration of applications and components. It handles snapshots, environments, pipelines and deployments, ensuring applications run efficiently and are deployed correctly.
+
+
+## Happy Path Tests.
+
+### Basic Integration Tests:
+
+- Testing for successful creation of applications and components.
+- Asserting the successful creation of a snapshot after a push event.
+- Checking if the BuildPipelineRun is successfully triggered and completed.
+- Asserting the signing of BuildPipelineRun.
+- Validating the successful creation of a SnapshotEnvironmentBinding.
+
+
+### Pipeline and Integration Scenarios:
+
+- Verifying that all integration pipeline runs finish successfully.
+- Checking that a snapshot gets successfully marked as 'passed' after specific events.
+
+
+### Environment and Deployment Scenarios:
+
+- Ensuring that environments are created successfully.
+- Verifying the existence of deploymentTargetClass.
```

Comment:
If the ephemeral env test case added, we need a deploymentTargetClass CR.  But we don't need to verify it.